string object_type
---
asr_msgs/AsrViewport[] viewport_list